src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+
+const FEMALE = 'f';
```

Comment:
Move it to separate file and reuse in all code

Reply:
Not fixed 😢